index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+var bodyParser = require('body-parser');
+var express = require('express');
+var gitHubWebHook = require('express-github-webhook');
+var nconf = require('nconf');
+var Promise = require('bluebird');
+
+var GitHubServer = require('./lib/GitHubServer');
+var getGoogleGroupLinks = require('./lib/getGoogleGroupLinks');
+
+var app = express();
+module.exports = app;
+
+nconf.env('__')
+    .file({
+        file: 'config.json'
+    });
+
+nconf.defaults({
+    port: 5000,
+    secret: '', // Repository secret to verify incoming WebHook requests from GitHub
+    gitHubToken: '', // Token used to verify outgoing requests to GitHub repository
+    repository: '', // Repository to scan for outdated pull requests and bump them
+    listenPath: '/' // Path on which to listen for incoming requests
+});
+
+var webHookHandler = gitHubWebHook({
+    path: nconf.get('listenPath'),
+    secret: nconf.get('secret')
+});
+
+app.use(bodyParser.json());
+app.use(webHookHandler);
+
+var gitHubServer = new GitHubServer('cesium-concierge', nconf.get('gitHubToken'));
+
+/** Get comments -> regex search -> post comment
+ *
+ * @param {String} commentsUrl URL to set/get comments on issue (https://developer.github.com/v3/activity/events/types/#issuesevent)
+ * @return {Promise<http.IncomingMessage>} Response
+ */
+function commentOnClosedIssue(commentsUrl) {
```

Comment:
Although, looking at GithubServer, I'm not sure it needs to exist.  Aren't most of these functions just isolated?  Would it be better to have a pile of functions instead of a `GithubServer` object?